lib/client/src/client.rs

```diff
@@ -428,13 +450,21 @@ impl MangoClient {
     ) -> anyhow::Result<PreparedInstructions> {
         let token = self.context.token_by_mint(&mint)?;
         let token_index = token.token_index;
+        let fallback_contexts = self
+            .context
+            .derive_fallback_oracle_keys(
+                &self.client.fallback_oracle_config,
+                &self.client.rpc_async(),
+            )
+            .await?;
 
         let (health_check_metas, health_cu) =
             self.context.derive_health_check_remaining_account_metas(
```

Comment:
this sounds like there should be a new helper function for derive_health_check_remaining_account_metas in the MangoClient (similar how there's one for the health_check remainig account metas), that can hide fetching the fallback_contexts. Otherwise that needs to be repeated everywhere.